DigitalOffice/Services/AuthenticationService/Validators/UserLoginValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.AuthenticationService.Models;
+
+namespace LT.DigitalOffice.AuthenticationService.Validators
+{
+    public class UserLoginValidator : AbstractValidator<UserCredentialsRequest>
+    {
+        public UserLoginValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .MaximumLength(164)
+                .WithMessage("Email is too long.")
+                .EmailAddress()
+                .WithMessage("Email is invalid.");
+
+            RuleFor(user => user.Password)
+                .NotEmpty()
+                .WithMessage("Password is empty.");
```

Comment:
Remove it message.

Reply:
Why?

Reply:
should we not understand why the data did not pass validation?

Reply:
https://digitaloffice-group.slack.com/archives/C016H4Z2LGL/p1595663778001900

Reply:
Fixed